client/src/TitleScreen/TitleScreen.js

```diff
@@ -192,3 +199,14 @@ async function setUser(body) {
 	});
 	return response;
 }
+
+// Mute a singular HTML5 element
+function muteMe(elem, muteToggle) {
+	elem.muted = muteToggle;
+    elem.pause();
+}
+// Try to mute all video and audio elements on the page
+function mutePage() {
```

Comment:
Then you need to accept the parameter

```suggestion
function mutePage(muteToggle) {
```